buildrunner/utils.py

```diff
@@ -20,6 +23,13 @@
 
 from buildrunner.errors import BuildRunnerConfigurationError
 
+LOCK_TIMEOUT_SECONDS = 600.0
+
+class FailureToAcquireLockException(Exception):
```

Comment:
This seems dumb, but there should be two spaces before a class or a method. This applies to everything below as well. pycodestyle usually catches this...

Reply:
I will make that change. There is no `pycodestyle` in `buildrunner`.  I would add it in this PR but there are lots of places that would need to be fixed. So if we want to add it it should be a separate PR IMO. I'll make a ticket and we can discuss it.

Reply:
Agreed, thanks